webui/react/src/hooks/useModal/Model/useModalModelVersionDelete.tsx

```diff
@@ -54,24 +46,22 @@ const useModalModelVersionDelete = ({ onClose }: Props = {}): ModalHooks => {
 
   const getModalProps = useCallback(
     (modelVersion: ModelVersion): ModalFuncProps => {
-      return canDeleteModelVersion({ modelVersion })
-        ? {
-            closable: true,
-            content: `
+      return {
+        closable: true,
+        content: `
         Are you sure you want to delete this version
         "Version ${modelVersion.version}" from this model?
       `,
-            icon: null,
-            maskClosable: true,
-            okButtonProps: { type: 'primary' },
-            okText: 'Delete Version',
-            okType: 'danger',
-            onOk: () => handleOk(modelVersion),
-            title: 'Confirm Delete',
-          }
-        : clone(CANNOT_DELETE_MODAL_PROPS);
+        icon: null,
+        maskClosable: true,
+        okButtonProps: { type: 'primary' },
+        okText: 'Delete Version',
+        okType: 'danger',
+        onOk: () => handleOk(modelVersion),
```

Comment:
nit: `handleOk` func can be inside `getModalProps` func, then we can remove dependencies and easier to follow the code imo

Reply:
✅